.github/workflows/dependabot_update_rbis.yml

```diff
@@ -0,0 +1,57 @@
+name: Update gem RBIs on Dependabot PRs
+
+on: workflow_call
+
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  update-gem-rbis:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
```

Comment:
Nit: Timeout can be significantly reduced given the average `bundle install` time on ruby-lsp. 10-15 mins would be enough buffer.

Reply:
👍 Other projects may need be a bit longer than Ruby LSP, but 15m seems plenty, updated.